components/serverless/pkg/apis/serverless/v1alpha2/function_types.go

```diff
@@ -50,6 +50,8 @@ type Source struct {
 
 type InlineSource struct {
 	// Specifies the Function's full source code.
+	// +kubebuilder:validation:Required
```

Comment:
Do you know why this requirement is not reflected in the CRD? 🤔 I can see only the `MinLength=1` is represented by `minLength: 1`.

Maybe CRD is not regenerated after changes or maybe it should looks like this and it's fine

Reply:
I know. `Required` is default value and is not necessary. I added it for better readibility and consistency with other fields (it's hard because we have mix - sometimes there is `Required` and sometimes not)

Reply:
ahhhh got it, thanks for the explanation